### PR TITLE
{chem}[intel-compilers/2022.1.0] json-fortran v8.3.0, mctc-lib v0.3.1, mstore v0.2.0 

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
After some discussion on the Slack channel, that is the problem I am facing. 
The previously used `intel-2022b` toolchain caused segfaults with tblite (PR #19224) and also CREST, so I wanted to revert back to `intel-2022a` as suggested by the developers of [CREST](https://github.com/crest-lab/crest/issues/238) 
Whereas for the `intel-2022b` toolchain, the correct modules are picked, i.e.:
```
== 2023-11-21 11:00:30,629 modules.py:626 INFO Checking whether mctc-lib/0.3.1-intel-compilers-2022.2.1 exists...
== 2023-11-21 11:00:30,629 modules.py:631 INFO Module mctc-lib/0.3.1-intel-compilers-2022.2.1 exists (found in list of available modules)
== 2023-11-21 11:00:30,629 modules.py:654 INFO Result for existence check of mctc-lib/0.3.1-intel-compilers-2022.2.1 module: True
```
that does not happen when I am downgrading to `intel-2022a`:
```
== 2023-11-22 10:00:49,383 modules.py:626 INFO Checking whether mctc-lib/0.3.1-GCCcore-11.3.0 exists...
== 2023-11-22 10:00:49,383 modules.py:631 INFO Module mctc-lib/0.3.1-GCCcore-11.3.0 exists (found in list of available modules)
== 2023-11-22 10:00:49,383 modules.py:654 INFO Result for existence check of mctc-lib/0.3.1-GCCcore-11.3.0 module: True
```
Given that all I done was a downgrade in the EasyConfig file, I am not sure where I am going wrong here. 
I somehow seem to miss something but I cannot find it.

**Comment by sassy-crick** [comment]
Thanks to the help fo @ocaisa the problems were fixed (see PR #19283) so that can be looked at.

File: easybuild/easyconfigs/j/json-fortran/json-fortran-8.3.0-intel-compilers-2022.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    ('binutils', '2.38'),
+    ('CMake', '3.24.3'),
+]
+
+configopts = '-DUSE_GNU_INSTALL_CONVENTION=TRUE'
+
+runtest = 'check'
+
+sanity_check_paths = {
+    'files': ['lib/libjsonfortran.a', 'lib/libjsonfortran.%s' % SHLIB_EXT],
```
Comment (ocaisa) [inline easybuild/easyconfigs/j/json-fortran/json-fortran-8.3.0-intel-compilers-2022.1.0.eb:27]:
```suggestion
    'files': ['lib/libjsonfortran.a', 'lib/libjsonfortran.%s' % SHLIB_EXT,
              'include/json_module.mod', 'include/json_parameters.mod'],
```

**Comment by sassy-crick** [comment]
@boegelbot please test @ generoso

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/akesandgren/b78270bba6bb458ddf063ce7906480bc for a full test report.

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**